### ISSUE TITLE
Update noVNC version

### ISSUE THREAD

**Issue by AntonAndreevichMoroz** [Ylianst/MeshCentral#4928]
We kindly ask you to update the version of noVNC you are using if possible.  Because there are interesting innovations.

https://github.com/novnc/noVNC/releases/tag/v1.4.0

**Comment by si458** [comment]
#5220 please can you try my PR ?

**Comment by AntonAndreevichMoroz** [comment]
> #5220 please can you try my PR ?

@si458 Thank you for pulling me, without this I would have gotten to this problem for a long time. I tried your PR, unfortunately it didn't work in default. I found the edits that were made to MeshCentral on the 1.3.0 example, added it to version 1.4.0 and it worked. I also checked the work of ZRLE coding, and it also works. I'll make a pull request now.
